docs/FAQ.md

```diff
@@ -30,16 +30,29 @@ to set up testing and validation on their distributions.
 
 ### What CUDA versions are supported?
 
-CUDA 10.1, 10.2 and 11.0 are currently supported, but you need to download the cudf jar that 
-corresponds to the version you are using. Please look [here](download.md) for download 
-links for the latest release.
+CUDA 11.0 and 11.2 are currently supported.  Please look [here](download.md) for download links for
+the latest release.
+
+### What hardware is supported? 
+
+The plugin is tested and supported on V100, T4, A30 and A100 Datacenter (Tesla) GPUs.  It is
```

Comment:
nit: This is confusing because there was a Tesla micro architecture released in 2007-2008, but I guess the NVIDIA marketing people ran out of names? I personally would just drop the Tesla part all together.

Reply:
Updated